sample/kotlin/src/main/kotlin/com/datadog/android/sample/picture/PictureViewModel.kt

```diff
@@ -40,7 +40,7 @@ internal class PictureViewModel : ViewModel() {
     }
 
     companion object {
-        private const val RANDOM_URL = "https://source.unsplash.com/random/800x450"
+        private const val RANDOM_URL = "https://picsum.photos/800/450"
         private const val FAILING_URL = "https://s0urce.unsplash.com/random/800x450"
```

Comment:
I guess you need to modify also the failing URL to keep it consistent ?

Reply:
I thought about leaving it since one failing url is as good as another, but you're right - I'll change it to maintain consistency